openbb_platform/platform/core/openbb_core/api/rest_api.py

```diff
@@ -10,31 +10,41 @@
 from openbb_core.api.router.system import router as router_system
 from openbb_core.app.model.abstract.error import OpenBBError
 from openbb_core.app.service.auth_service import AuthService
+from openbb_core.app.service.system_service import SystemService
 from openbb_core.app.version import VERSION
```

Comment:
Now that we have `system` in the module, we can remove this import and access the `system.version`, wdyt?